bin/run-e2e-tests.sh

```diff
@@ -0,0 +1,13 @@
+# Exit if any command fails
+set -e
+
+# Change to the expected directory
+cd "$(dirname "$0")"
+
+# Setup local environement
+./setup-local-env.sh
```

Comment:
you could run this in a subshell to avoir having to get back to your working directory (`cd` at line 11):

```
( ./setup-local-env.sh )
```